File: app/src/main/java/com/stevenschoen/putionew/model/responses/CreateZipResponse.java
package com.stevenschoen.putionew.model.responses;

import com.google.gson.annotations.SerializedName;
import com.stevenschoen.putionew.model.ResponseOrError;

public class CreateZipResponse extends ResponseOrError.BasePutioResponse {
  @SerializedName("zip_id")
  private long zipId;

  public long getZipId() {
    return zipId;
  }
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/CreateZipResponse.kt
package com.stevenschoen.putionew.model.responses

import com.google.gson.annotations.SerializedName
import com.stevenschoen.putionew.model.ResponseOrError

class CreateZipResponse(
    @SerializedName("zip_id")
    val zipId: Long
) : ResponseOrError.BasePutioResponse()
